docs/src/pages/en/responsive-controls.mdx

```diff
@@ -0,0 +1,441 @@
+---
+title: Responsive controls
+description: Learn how to make controls responsive with Media Chrome
+layout: ../../layouts/MainLayout.astro
+---
+
+import SandpackContainer from "../../components/SandpackContainer.astro";
+
+To provide the best user experience across the myriad of devices available in today's world, it's a good idea to modify your video player UI to adapt to different screen sizes.
+
+For example, smaller screens have less available space to display your UI, and cannot effectively use all of the controls that Media Chrome provides. There are two ways you can address responsive styling using Media Chrome, detailed below.
+
+## Using breakpoints
+
+Media Chrome helps you to make changes to your video player layout at different screen sizes by providing **breakpoints** for common device screen widths. Let's take a look at how you can use these breakpoints to style your video player at different widths.
+
+<SandpackContainer
+  stacked
+  reversed
+  height={770}
+  editorHeight={300}
+  hiddenCss={`
+    media-controller {
+      aspect-ratio: 16 / 9;
+    }`
+  }
+  active="css"
+  html={`<media-controller>
+  <video
+    slot="media"
+    src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/low.mp4"
+    playsinline
+    muted
+  ></video>
+  <media-play-button></media-play-button>
+  <div id="log"></div>
+</media-controller>
+`}
+  css={`media-controller {
+  --media-primary-color: red;
+}
+media-controller[breakpointsm] {
+  --media-primary-color: orange;
+}
+media-controller[breakpointmd] {
+  --media-primary-color: yellow;
+}
+media-controller[breakpointlg] {
+  --media-primary-color: green;
+}
+media-controller[breakpointxl] {
+  --media-primary-color: blue;
+}
+media-controller #log::before {
+  content: 'none';
+  display: flex;
+  color: #666;
+  width: 24px;
+  height: 24px;
+  border-radius: 100%;
+  align-items: center;
+  justify-content: center;
+  padding: 1em;
+  background-color: red;
+  font-family: sans-serif;
+  position: absolute;
+  right: 5px;
+  bottom: 5px;
+}
+media-controller[breakpointsm] #log::before {
+  content: 'sm';
+  background-color: orange;
+}
+media-controller[breakpointmd] #log::before {
+  content: 'md';
+  background-color: yellow;
+}
+media-controller[breakpointlg] #log::before {
+  content: 'lg';
+  background-color: green;
+}
+media-controller[breakpointxl] #log::before {
+  content: 'xl';
+  background-color: blue;
+}
+  `}
+/>
+
```

Comment:
suggestion(non-blocking): Mention that they can resize the page or open the example in Code Sandbox to see the different breakpoints in action.